### PR TITLE
Add wandb server debug command

### PR DESCRIPTION
**Pull request by ethanho154**
Fixes WB-NNNN
Fixes #NNNN

Description
-----------
What does the PR do?

Add `wandb server debug` to pull basic info about the deployment and download the debug bundle.

Testing
-------
How was this PR tested?

Checklist
-------
- Include reference to internal ticket "Fixes WB-NNNN" (and github issue "Fixes #NNNN" if applicable)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -12,3 +12,4 @@ PyYAML
 pathtools # supports vendored version of watchdog 0.9.0
 setproctitle
 setuptools
+pyjwt
```
Comment (dmitryduev) [inline requirements.txt:15]: We are trying to reduce the number of base dependencies as much as possible..Could you replace this with a lazy load and a prompt to install pyjwt when neede? @kptkin do you agree?

File: wandb/cli/cli.py [hunk 2]
```diff
@@ -1436,6 +1439,70 @@ def server():
     pass
 
 
+@server.command(context_settings=RUN_CONTEXT, help="Debug a local W&B server")
+@click.option("--host", default=None, help="Debug a specific instance of W&B")
+@click.option("--output", default=None, help="Debug bundle destination")
+@display_error
+def debug(host, output):
```
Comment (dmitryduev) [inline wandb/cli/cli.py:1446]: Could you please write a test for this?

**Review by dmitryduev — COMMENTED** [review]
Thank you @ethanho154!

File: requirements.txt (hunk 1 above)
Comment (ethanho154) [reply requirements.txt:15]: Hey all, thanks for checking this! I'm actually going to remove this requirement. Originally I was hitting the system-admin/api/env endpoint and using that pyjwt to decode the license from the debug bundle. But based on [comments on this pull request](https://github.com/wandb/core/pull/9826) I shouldn't be exposing that at all and will be using a redacted bundle which doesn't include the license so no need to decode.

**Pull request closed without merging**